### ISSUE TITLE
Question about loss function when training?

### ISSUE THREAD

**Issue by Hereiszjy** [hpcaitech/ColossalAI#4761]
I am training an image classfication model and ues two loss functions, just like nn.CrossEntropyLoss() and metrics loss (CircleLoss). How to initialize these two criterions and train?

criterionA = nn.CrossEntropyLoss()
criterionB = CircleLoss()
engine, train_dataloader, test_dataloader, _ = colossalai.initialize(model,
                                                                    optimizer,
                                                                    criterion,
                                                                    train_dataloader,
                                                                    test_dataloader)

**Comment by flybird11111** [comment]
Hello, we recommend using the Booster API, and the usage of 'initialize' will be deprecated. https://colossalai.org/docs/basics/booster_api， The definition of the 'criterion' function can be referenced in this example.https://github.com/hpcaitech/ColossalAI/blob/main/examples/images/vit/vit_train_demo.py

**Comment by Hereiszjy** [comment]
邮件已收到，辛苦

**Comment by Hereiszjy** [comment]
thanks
